symbolic-base/src/ZkFold/Base/Protocol/Protostar/Fold.hs

```diff
@@ -138,14 +138,15 @@ ivcVerifierAc
     => Support y ~ Proxy ctx
     => Representable (Layout i)
     => Representable (Layout c)
-    => Representable (Layout f)
+    -- => Representable (Layout f)
```

Comment:
These are required to compile `ivcVerifier` into circuit, better leave them as is

Reply:
I reverted these comments and merged in changes from main. However, this code fails in a way that is a bit inscrutable to me, subject to whether or not a `4`-tuple has hand-rolled or generically-derived `SymbolicData` instance. Any insight would be appreciated.

Reply:
That's unfortunate, I think we need to reconstruct `t` from scratch for this to work
And let's wait for #325 to merge

Reply:
refactored `t`